#
# Make sure FMmpeg is not downloaded when not found.
#
--- a/VDF.GUI/ViewModels/MainWindowVM.cs
+++ b/VDF.GUI/ViewModels/MainWindowVM.cs
@@ -1287,6 +1287,7 @@ Non-Windows setup:
 				}
 			}
 
+			/* Do not download FFmpeg automatically.
 			if ((SettingsFile.Instance.UseNativeFfmpegBinding && !ScanEngine.NativeFFmpegExists) ||
 				(!SettingsFile.Instance.UseNativeFfmpegBinding && !ScanEngine.FFmpegExists) ||
 				!ScanEngine.FFprobeExists) {
@@ -1297,6 +1298,7 @@ Non-Windows setup:
 				await MessageBoxService.Show(GetRequiredFfmpegPackage(CoreUtils.CurrentFolder));
 				return;
 			}
+			*/
 			if (!ScanEngine.FFprobeExists) {
 				if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
 					await MessageBoxService.Show(App.Lang["Message.FFprobeMissingWithHint"]);
