--- a/VDF.GUI/Views/AboutWindow.xaml
+++ b/VDF.GUI/Views/AboutWindow.xaml
@@ -49,10 +49,12 @@
                 x:Name="ProjectPageButton"
                 Margin="4"
                 Click="OnProjectPage" />
+<!-- No need to show the "Latest Release" button: cannot launch URL in Docker container.
             <Button
                 x:Name="LatestReleaseButton"
                 Margin="4"
                 Click="OnLatestRelease" />
+-->
             <Button
                 x:Name="CopyButton"
                 Margin="4"
