diff --git a/VDF.GUI/Views/MainWindow.xaml b/VDF.GUI/Views/MainWindow.xaml
index 33cea38..9591e97 100644
--- a/VDF.GUI/Views/MainWindow.xaml
+++ b/VDF.GUI/Views/MainWindow.xaml
@@ -1397,11 +1397,13 @@
                                                         BorderThickness="1"
                                                         DragDrop.AllowDrop="True"
                                                         ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Includes}" />
+<!-- Drag and drop not supported with Docker container.
                                                     <TextBlock
                                                         Grid.Row="2"
                                                         Margin="5"
                                                         Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                         Text="Hint: You can also add folders with drag &amp; drop" />
+-->
                                                 </Grid>
                                                 <TextBlock
                                                     Grid.Row="0"
@@ -1467,11 +1469,13 @@
                                                         BorderThickness="1"
                                                         DragDrop.AllowDrop="True"
                                                         ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Blacklists}" />
+<!-- Drag and drop not supported with Docker container.
                                                     <TextBlock
                                                         Grid.Row="2"
                                                         Margin="5"
                                                         Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                         Text="Hint: You can also add folders with drag &amp; drop" />
+-->
                                                 </Grid>
                                             </Grid>
                                         </Grid>
