--- a/VDF.GUI/Views/MainWindow.xaml
+++ b/VDF.GUI/Views/MainWindow.xaml
@@ -1335,11 +1335,13 @@
                                                         BorderThickness="1"
                                                         DragDrop.AllowDrop="True"
                                                         ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Includes}" />
+<!-- Drag and drop not supported with Docker container.
                                                     <TextBlock
                                                         Grid.Row="2"
                                                         Margin="5"
                                                         Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                         Text="{Binding Source={x:Static local:App.Lang}, Path=[MainWindow.Settings.DragDropHint]}" />
+-->
                                                 </Grid>
                                                 <TextBlock
                                                     Grid.Row="0"
@@ -1405,11 +1407,13 @@
                                                         BorderThickness="1"
                                                         DragDrop.AllowDrop="True"
                                                         ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Blacklists}" />
+<!-- Drag and drop not supported with Docker container.
                                                     <TextBlock
                                                         Grid.Row="2"
                                                         Margin="5"
                                                         Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                         Text="{Binding Source={x:Static local:App.Lang}, Path=[MainWindow.Settings.DragDropHint]}" />
+-->
                                                 </Grid>
                                             </Grid>
                                         </Grid>
--- a/VDF.GUI/Views/ChooseAlgoView.axaml
+++ b/VDF.GUI/Views/ChooseAlgoView.axaml
@@ -81,11 +81,13 @@
 							MinHeight="200"
 							ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Includes}"
 							SelectionMode="Multiple" />
+<!-- Drag and drop not supported with Docker container.
 							<TextBlock
 							Grid.Row="2"
 							Margin="4,6,0,0"
 							Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
 							Text="{Binding Source={x:Static app:App.Lang}, Path=[MainWindow.Settings.DragDropHint]}" />
+-->
 						</Grid>
 					</StackPanel>
 				</Border>
