--- a/VDF.GUI/Views/MainWindow.xaml
+++ b/VDF.GUI/Views/MainWindow.xaml
@@ -1658,21 +1658,27 @@
                                             Text="Appearance" />
                                     </TabItem.Header>
                                     <StackPanel>
+<!-- Windows-only setting never applies with a Docker container.
                                         <CheckBox
                                             VerticalAlignment="Center"
                                             Content="Use Mica (Windows 11 only)¹"
                                             IsChecked="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=UseMica}"
                                             IsThreeState="False" />
+-->
+<!-- Dark mode is handled by the Docker container.
                                         <CheckBox
                                             VerticalAlignment="Center"
                                             Content="Dark Mode¹"
                                             IsChecked="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=DarkMode}"
                                             IsThreeState="False" />
+-->
+<!-- This text is not needed since all relevant settings are hidden with a Docker container.
                                         <TextBlock
                                             Margin="0,5,0,5"
                                             VerticalAlignment="Center"
                                             Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                             Text="¹ Restart VDF to apply changes" />
+-->
                                         <CheckBox
                                             VerticalAlignment="Center"
                                             Content="Show Thumbnail Column"
