--- a/VDF.GUI/Views/MainWindow.xaml
+++ b/VDF.GUI/Views/MainWindow.xaml
@@ -1494,8 +1494,10 @@
                                                      DragDrop.AllowDrop="True"
                                                      ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Includes}" />
                                             <StackPanel Grid.Row="2">
+<!-- Drag and drop not supported with Docker container.
                                                 <TextBlock Margin="5" TextWrapping="Wrap" Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                            Text="{Binding Source={x:Static local:App.Lang}, Path=[MainWindow.Settings.DragDropHint]}" />
+-->
                                                 <TextBlock Margin="5" TextWrapping="Wrap" Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                            Text="{Binding Source={x:Static local:App.Lang}, Path=[MainWindow.Settings.WildcardNote]}" />
                                             </StackPanel>
@@ -1564,8 +1566,10 @@
                                             <ListBox Name="ListboxBlacklist" Grid.Row="1" BorderThickness="1"
                                                      DragDrop.AllowDrop="True"
                                                      ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Blacklists}" />
+<!-- Drag and drop not supported with Docker container.
                                             <TextBlock Grid.Row="2" Margin="5" TextWrapping="Wrap" Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                        Text="{Binding Source={x:Static local:App.Lang}, Path=[MainWindow.Settings.DragDropHint]}" />
+-->
                                             <TextBlock Grid.Row="3" Margin="5" TextWrapping="Wrap" Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
                                                        Text="{Binding Source={x:Static local:App.Lang}, Path=[MainWindow.Settings.WildcardNote]}" />
                                         </Grid>
--- a/VDF.GUI/Views/ChooseAlgoView.axaml
+++ b/VDF.GUI/Views/ChooseAlgoView.axaml
@@ -81,11 +81,13 @@
 							MinHeight="200"
 							ItemsSource="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=Includes}"
 							SelectionMode="Multiple" />
+<!-- Drag and drop not supported with Docker container.
 							<TextBlock
 							Grid.Row="2"
 							Margin="4,6,0,0"
 							Foreground="{actipro:ThemeResource ControlBackgroundBrushSolidWarning}"
 							Text="{Binding Source={x:Static app:App.Lang}, Path=[MainWindow.Settings.DragDropHint]}" />
+-->
 						</Grid>
 					</StackPanel>
 				</Border>
