--- a/VDF.Core/VDF.Core.csproj
+++ b/VDF.Core/VDF.Core.csproj
@@ -9,7 +9,7 @@
     <InternalsVisibleTo Include="VDF.GUI" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="FFmpeg.AutoGen" Version="6.0.0" />
+    <PackageReference Include="FFmpeg.AutoGen" Version="5.1.2.3" />
     <PackageReference Include="Mono.Posix.NETStandard" Version="5.20.1-preview" />
     <PackageReference Include="protobuf-net" Version="3.2.16" />
     <PackageReference Include="SixLabors.ImageSharp" Version="1.0.4" />
